ega-data-api-key/src/main/java/eu/elixir/ega/ebi/keyproviderservice/CustomAutoconfiguration.java

```diff
@@ -0,0 +1,30 @@
+package eu.elixir.ega.ebi.keyproviderservice;
+
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
+import org.springframework.boot.autoconfigure.jdbc.DataSourceTransactionManagerAutoConfiguration;
+import org.springframework.boot.autoconfigure.orm.jpa.HibernateJpaAutoConfiguration;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+
+@Configuration
+public class CustomAutoconfiguration {
```

Comment:
I tried moving it inside the config package but then could not able to fix the bean error for profile 'db' so sure how to do it.

Reply:
Could you send me the error via slack? Or maybe just some config example?

Reply:
Ahh I just tried with some config from the it tests:

You could add @EntityScan and @EnableJpaRepositories so that spring knows where to find the entities and repositories.

``` 
   @Profile("db")
  @Configuration
  @EnableAutoConfiguration
  @EntityScan(basePackages = "eu.elixir.ega.ebi.keyproviderservice.domain.entity" )
  @EnableJpaRepositories(basePackages = "eu.elixir.ega.ebi.keyproviderservice.domain.repository" )
  public class DbAutoconfiguration {

  }
```